src/python/elements.cpp

```diff
@@ -810,7 +809,16 @@ void init_elements(py::module& m)
                      std::make_pair("psi", dip_edge.m_psi),
                      std::make_pair("rc", dip_edge.m_rc),
                      std::make_pair("g", dip_edge.m_g),
-                     std::make_pair("K2", dip_edge.m_K2)
+                     std::make_pair("R", dip_edge.m_R),
+                     std::make_pair("K0", dip_edge.m_K0),
+                     std::make_pair("K1", dip_edge.m_K1),
+                     std::make_pair("K2", dip_edge.m_K2),
+                     std::make_pair("K3", dip_edge.m_K3),
+                     std::make_pair("K4", dip_edge.m_K4),
+                     std::make_pair("K5", dip_edge.m_K5),
+                     std::make_pair("K6", dip_edge.m_K6)
+                     //std::make_pair("model", amrex::getEnumNameString(dip_edge.m_model)),
+                     //std::make_pair("location", amrex::getEnumNameString(dip_edge.m_location))
```

Comment:
@WeiqunZhang , I do not understand why I cannot do:
```C++
std::string model_str = amrex::getEnumNameString(dip_edge.m_model);
```

It does not compile although `dip_edge.m_model` is an `AMREX_ENUM`.